admin/tools/docker/deployment/in2p3/examples/docker-status.sh

```diff
@@ -0,0 +1,4 @@
+set -x
+
+/opt/shmux/bin/shmux -c "echo \"$PASSWORD\" | sudo -S sh -c 'service docker status'" ccqserv{125..149}.in2p3.fr
+
```

Comment:
I don't understand why some of these include all nodes and others only include the upper 50 nodes..